workers/unity/Assets/Playground/Scripts/Worker/IMobileClient.cs

```diff
@@ -0,0 +1,11 @@
+namespace Playground
+{
+    public interface IMobileClient
```

Comment:
would change that name to probably something like `MobileConnectionController`?
Not sure what would be a good name, but `IMobileClient` seems to broad